NOTICE.txt

```diff
@@ -0,0 +1,21 @@
+This product includes software from the twox-hash project
+ * Copyright https://github.com/shepmaster/twox-hash
+ * Licensed under the MIT License;
```

Comment:
hmmm so does this mean that comet would be dual licensed?

Not sure the legal part... Especially the Copyright github url part...

Reply:
It would be good to get another opinion on this though. Perhaps @alamb could offer some thoughts.